packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimeRangeList.tsx

```diff
@@ -73,12 +73,19 @@ function isEqual(x: TimeOption, y?: TimeOption): boolean {
   return y.from === x.from && y.to === x.to;
 }
 
-const getStyles = () => ({
+const getStyles = (theme: GrafanaTheme2) => ({
   title: css({
     display: 'flex',
     alignItems: 'center',
     justifyContent: 'space-between',
     padding: '8px 16px 5px 9px',
+
+    '+ ul': {
+      [theme.breakpoints.up('lg')]: {
+        height: '116px',
```

Comment:
what happens in the future if we change the height of the picker? or add another `ul` element? 🤔 
(let's try and avoid magic numbers and selecting specific elements via generic element types)